tensorboard/plugins/hparams/summary_v2_test.py

```diff
@@ -512,20 +471,25 @@ def test_backward_endpoints(self):
 
   def test_sample_uniform(self):
     domain = hp.IntInterval(2, 7)
-    rng = random.Random(0)
-    mu = 10000
-    trials = (domain.max_value - domain.min_value + 1) * mu
-    samples = [domain.sample_uniform() for _ in xrange(trials)]
-    range_ = xrange(domain.min_value, domain.max_value + 1)
-    assert_uniform(self, range_, samples)
+    rng = mock.Mock()
+    sentinel = object()
+    # Note: `randint` samples from a closed interval, which is what we
+    # want (as opposed to `randrange`).
+    rng.randint.return_value = sentinel
+    result = domain.sample_uniform(rng)
+    self.assertIs(result, sentinel)
+    rng.randint.assert_called_once_with(2, 7)
 
   def test_sample_uniform_unseeded(self):
     domain = hp.IntInterval(2, 7)
-    for _ in xrange(100):
-      sample = domain.sample_uniform()
-      self.assertIsInstance(sample, int)
-      self.assertGreaterEqual(sample, domain.min_value)
-      self.assertLessEqual(sample, domain.max_value)
+    # Note: `randint` samples from a closed interval, which is what we
+    # want (as opposed to `randrange`).
+    with mock.patch("random.randint") as m:
```

Comment:
Slightly cleaner IMO to `import random` and then use `mock.patch.object` (this is what the internal mocking guide recommends) but up to you.

Reply:
Done; thanks for the reference.